src/ck-ldb/HybridBaseLB.h

```diff
@@ -379,10 +379,10 @@ class HybridBaseLB : public CBase_HybridBaseLB
     int info_recved;		// for CollectInfo()
     int vector_expected, vector_completed;
     int resumeAfterMigration;
-    CkVec<MigrationRecord> outObjs;
-    //CkVec<Location> unmatchedObjs;
+    std::vector<MigrationRecord> outObjs;
+    //std::vector<Location> unmatchedObjs;
     std::map< LDObjKey, int >  unmatchedObjs;
```

Comment:
Could this be unordered_map?

Reply:
Yeah, but that's out of scope for this change, I think; we'd need to add hashing for `LDObjKey`, so it's not a trivial change.